<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sender</class>
 <widget class="QWidget" name="Sender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sender</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addPort">
        <property name="text">
         <string>Add Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removePort">
        <property name="text">
         <string>Remove Port</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="portView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
